mbedtls/src/lib.rs

```diff
@@ -7,6 +7,7 @@
  * according to those terms. */
 
 #![deny(warnings)]
+#![allow(dead_code)]
```

Comment:
Why?

Reply:
Some variables are only set and never read. 

I moved from this define to those variables -> see mbedtls/src/ssl/config.rs:122

```
    #[allow(dead_code)]
    dhm: Option<Arc<Dhm>>,
```